src/modules/elements/creatable-select/style.pcss

```diff
@@ -32,6 +32,11 @@
 		& > input {
 			margin: 0;
 			line-height: 1.5;
+
+			&,
```

Comment:
Is this needed?

Reply:
there's a transparent box shadow on the input when not focused, so not necessarily needed, but added it to remove all box shadows.